src/main/resources/avro/bdg.avdl

```diff
@@ -472,44 +478,99 @@ enum GenotypeType {
-  union { null, float }   readPositionRankSum = null;
+
+  /**
+   The Wilcoxon rank-sum test statistic of the base quality scores. The base quality
+   scores are separated by whether or not the base supports the reference or the
+   alternate allele.
+   */
+  union { null, float } baseQRankSum = null;
+
+  /**
+   */
+  union { null, float } clippingRankSum = null;
+
+  /**
+   The Fisher's exact test score for the strand bias of the reference and alternate
+   alleles. Stored as a phred scaled probability. Thus, if:
+
+   * a = The number of positive strand reads covering the reference allele
+   * b = The number of positive strand reads covering the alternate allele
+   * c = The number of negative strand reads covering the reference allele
+   * d = The number of negative strand reads covering the alternate allele
+
+   This value takes the score:
+   
+   -10 log((a + b)! * (c + d)! * (a + c)! * (b + d)! / (a! b! c! d! n!)
+
+   Where n = a + b + c + d.
+   */
+  union { null, float } fisherStrandBiasPValue = null;
+
+  /**
+   The root mean square of the mapping qualities of reads covering this site.
+   */
+  union { null, float } rmsMapQ = null;
+  
+  /**
+   The number of reads at this site with mapping quality equal to 0.
+   */
+  union { null, int } mapq0Reads = null;
+
+  /**
+   The Wilcoxon rank-sum test statistic of the mqpping quality scores. The mapping
+   quality scores are separated by whether or not the read supported the reference or the
+   alternate allele.
+   */
+  union { null, float } mqRankSum = null;
+
+  /**
+   The Wilcoxon rank-sum test statistic of the position of the base in the read at this site.
+   The positions are separated by whether or not the base supports the reference or the
+   alternate allele.
+   */
+  union { null, float } readPositionRankSum = null;
 
   /**
    The Phred scaled prior probabilities of the various genotype states at this site.
    The number of elements in this array should be equal to the ploidy at this
    site, plus 1.
    */
-  array<int> genotypePriors = [];
+  array<float> genotypePriors = [];
+
   /**
    The Phred scaled posterior probabilities of the various genotype states at this site,
    in this sample. The number of elements in this array should be equal to the ploidy at
    this site, plus 1.
    */
-  array<int> genotypePosteriors = [];
+  array<float> genotypePosteriors = [];
 
-  // VQSR: Log-odds ratio of being a true vs false variant under trained
-  // Gaussian mixture model.
-  union { null, float }   vqslod = null;
-  union { null, string }  culprit = null;
+  /**
+    The log-odds ratio of being a true vs. false variant under a trained statistical model.
+    This model can be a multivariate Gaussian mixture, support vector machine, etc.
+    */
+  union { null, float } vqslod = null;
+
+  /**
+   If known, the feature that contributed the most to this variant being classified as
+   a false variant.
+   */
+  union { null, string } culprit = null;
```

Comment:
I believe the GATK does. It's useful if you want to "see" why VQSR is filtering variants out. E.g., if you see that VQSR is filtering most variants out on the basis of depth, you can just use a static read depth filter instead of training a statistical model on all your variant calls. We've got WIP VQSR code for avocado and I think we emit this field.